staff/sys/ocf-kubernetes-deploy

```diff
@@ -36,13 +44,20 @@ def main():
              "app-{appname}".
         ''',
     )
+
+    # This argument is optional because some Kubernetes services
+    # (sourcegraph) don't need us to build our own Docker images
     parser.add_argument(
         'appversion',
+        nargs='?',
+        default='',
```

Comment:
Have you tested the case where this isn't passed in? I believe `kubernetes-deploy` requires that the `version` variable is passed as a binding.

Reply:
I have tested this, kubernetes deploy doesn't require a version binding. That was the REVISION binding and is no longer mandatory.

Reply:
Oh cool that's good to know